tools/packaging/deploy/ccruntime.yaml

```diff
@@ -23,10 +23,6 @@ spec:
         name: enclave-cc-conf
       - mountPath: /opt/confidential-containers/
         name: enclave-cc-artifacts
-      - mountPath: /var/run/dbus
```

Comment:
`ccruntime.yaml` in this repo is unused so the changes can be dropped (unless you don't mind removing it in a separate commit here).

Reply:
@mythi, I've updated @jepio's PR to drop the file.